package.json

```diff
@@ -17,18 +16,18 @@
     "date-fns": "^2.25.0",
     "fs": "0.0.1-security",
     "keys": "^0.1.2",
-    "next": "12.1.0",
```

Comment:
It looks like `fs`, `keys` and `typescript` are not used, so we could remove them. Could you verify that?

Reply:
@orlyohreally 
I can't pull out `typescript` as that's a peer dependency for `ts-utils`: 
`tsutils@3.21.0 requires a peer of typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta but none is installed. You must install peer dependencies yourself.`

But I was able to pull out `fs` and `keys`. Has been redeployed to both heroku and vercel (links above0